tools/cli/pkg/ers/client/api.go

```diff
@@ -0,0 +1,13 @@
+package client
+
+import "github.com/kyma-project/control-plane/tools/cli/pkg/ers"
+
+type Client interface {
+	GetOne(id string) (*ers.Instance, error)
```

Comment:
Isn't `id` correct here? The mentioned section from [](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) states `Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url"`. So as long it is the same case it seems correct.

Reply:
"This rule also applies to "ID" when it is short for "identifier" (which is pretty much all cases when it's not the "id" as in "ego", "superego"), so write "appID" instead of "appId"."